tests/fe/util_test.py

```diff
@@ -12,35 +12,47 @@ def test_expires_when_str():
 
     assert expires_when_str(None) == "Never", "no datetime means no expires"
 
-    for date_, expected, msg in [
-        ("2015-08-11 11:00:00.000", "Expired", "long before should expire"),
-        ("2015-08-11 12:00:00.000", "Expired", "same time should expire"),
-        ("2015-08-11 11:59:59.000", "Expired", "minute after should expire"),
-        ("2015-08-11 12:00:00.100", "Expired", "milliseonds before should expire"),
```

Comment:
This case + the equivalent one in `test_long_ago_str` are a bit confusing without a comment somewhere on how we're (manually) only precise to the second, given that `datetime` and `relativedelta` on their own can handle the finer precision